### PR TITLE
Update `emit` to pass the event to `trigger`

### PR DESCRIPTION
**Pull request by jeffcole**
That way the emit handler can do things like stop propagation of the event to parent views, or `preventDefault` on it.

I made a `diamond-freight` branch with this change as well because `master` breaks things in that project.

On a fresh build the suite is red. Anyone know if that's expected?

### CODE REVIEW

File: coffee/src/rendering/nodeStrategy.coffee [hunk 1]
```diff
@@ -32,4 +32,4 @@ class Traction.Rendering.NodeStrategy extends Backbone.View
 
   _emit: (event) ->
     eventNames = event.currentTarget.getAttribute("data-emit").split(" ")
-    @$el.trigger(eventName) for eventName in eventNames
+    @$el.trigger(eventName, event) for eventName in eventNames
```
Comment (artfuldodger) [inline coffee/src/rendering/nodeStrategy.coffee]: For the record, this is the code change.

**Comment by artfuldodger** [comment]
What's with all the new spec files? Maybe would be nice to split CoffeeScript upgrade changes into separate commit, but :+1:

**Comment by jeffcole** [comment]
@artfuldodger hmmm looks like those may have been generated by my running the specs and doing `git add .`. I just did this whole thang fresh without doing so and looks like those files didn't get added.

Good call on separating out the build stuff, I did that. Thank yoooouuuu.

File: coffee/src/rendering/nodeStrategy.coffee (hunk 1 above)
Comment (jeffcole) [inline coffee/src/rendering/nodeStrategy.coffee]: :100:

Comment (jakesutton) [inline coffee/src/rendering/nodeStrategy.coffee]: Uh... Yeah... Okay?

**Comment by jakesutton** [comment]
The actual change seems fine. :+1:

File: coffee/src/rendering/nodeStrategy.coffee (hunk 1 above)
Comment (artfuldodger) [inline coffee/src/rendering/nodeStrategy.coffee]: It was more confusing before the unintentional spec files went away. :stuck_out_tongue:

**Pull request merged**